verification/cbmc/proofs/aws_hash_iter_delete/Makefile

```diff
@@ -17,12 +14,13 @@ PROOF_SOURCES += $(HARNESS_FILE)
 PROOF_SOURCES += $(PROOF_SOURCE)/make_common_data_structures.c
 PROOF_SOURCES += $(PROOF_SOURCE)/proof_allocators.c
 PROOF_SOURCES += $(PROOF_SOURCE)/utils.c
+PROOF_SOURCES += $(PROOF_STUB)/s_remove_entry_override.c
```

Comment:
This should be `ABSTRACTIONS` not proof sources.

Reply:
There are many files that use `PROOF_SOURCES` instead of `ABSTRACTIONS`.